mailbox/backup/src/test/java/org/apache/james/mailbox/backup/ZipperTest.java

```diff
@@ -113,4 +116,40 @@ void archiveShouldWriteSizeMetadata() throws Exception {
                         .containsExtraFields(new SizeExtraField(SIZE_1)));
         }
     }
+
+    @Test
+    void archiveShouldWriteUIDMetadata() throws Exception {
```

Comment:
Maybe `archiveShouldWriteUIDMetadata` `archiveShouldWriteMessageIdMetadata` `archiveShouldWriteMailboxIdMetadata` can be combined in a single test that  would assert all the properties: `archiveShouldWriteMailboxMessageMetadata`

BTW it should be archiveShouldWriteUidMetadata